Sources/Prometheus/MetricDescriptor.swift

```diff
@@ -30,7 +30,7 @@ public struct MetricNameDescriptor {
     /// An optional suffix describing the metric's unit (e.g., `total`).
     public let unitName: String?
 
-    /// Optional descriptive text for the metric.
+    /// Optional help text for the metric. If a non-empty string is provided, it will be emitted as a `# HELP` line in the exposition format. If the parameter is omitted or an empty string is passed, the `# HELP` line will not be generated for this metric.
```

Comment:
That's a very long line, can we add some linebreaks please?

Reply:
The formatting let it pass though, yup will do!